src/redux/reducers/index.js

```diff
@@ -0,0 +1,17 @@
+/* Here goes the simple index action creator for returning the welcoming message
```

Comment:
index.js is always a defaullt file of a folder. We can avoid using `index.js` and add it as another file e.g `default.js`. 
I could expect rootReducer to be in this file if we go with index.js.